### ISSUE TITLE
ember-cli 0.2.5 generator issue

### ISSUE THREAD

**Issue by maxkazar** [kimroen/ember-cli-coffeescript#81]
Try to 

```
ember g component users-toolbar -p -path users
```

and result 

```
Arguments to path.join must be strings
TypeError: Arguments to path.join must be strings
  at Object.posix.join (path.js:488:13)
  at __path__ (/Users/kazar/Projects/medmedia/mimic-ember/node_modules/ember-cli/blueprints/component/index.js:26:23)
  at /Users/kazar/Projects/medmedia/mimic-ember/node_modules/ember-cli/lib/models/blueprint.js:628:66
  at Array.map (native)
  at Class.Blueprint.generateFileMap (/Users/kazar/Projects/medmedia/mimic-ember/node_modules/ember-cli/lib/models/blueprint.js:628:37)
  at Class.Blueprint._locals (/Users/kazar/Projects/medmedia/mimic-ember/node_modules/ember-cli/lib/models/blueprint.js:814:22)
  at Class.Blueprint._process (/Users/kazar/Projects/medmedia/mimic-ember/node_modules/ember-cli/lib/models/blueprint.js:458:22)
  at Class.Blueprint.install (/Users/kazar/Projects/medmedia/mimic-ember/node_modules/ember-cli/lib/models/blueprint.js:491:15)
  at Class.module.exports.Task.extend.run (/Users/kazar/Projects/medmedia/mimic-ember/node_modules/ember-cli/lib/tasks/generate-from-blueprint.js:52:49)
  at Class.module.exports.Command.extend.run (/Users/kazar/Projects/medmedia/mimic-ember/node_modules/ember-cli/lib/commands/generate.js:79:17)
  at Class.<anonymous> (/Users/kazar/Projects/medmedia/mimic-ember/node_modules/ember-cli/lib/models/command.js:136:17)
  at lib$rsvp$$internal$$tryCatch (/Users/kazar/Projects/medmedia/mimic-ember/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:489:16)
  at lib$rsvp$$internal$$invokeCallback (/Users/kazar/Projects/medmedia/mimic-ember/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:501:17)
  at lib$rsvp$$internal$$publish (/Users/kazar/Projects/medmedia/mimic-ember/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:472:11)
  at lib$rsvp$asap$$flush (/Users/kazar/Projects/medmedia/mimic-ember/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:1290:9)
  at process._tickCallback (node.js:355:11)
```

After remove `ember-cli-coffeescript` all work fine

**Comment by artsyca** [comment]
I believe this also happens when you do `--pod`

**Comment by kimroen** [comment]
Hi!

Sorry that I haven't responded to this sooner. As far as I can tell, the problem is that the `-path` argument that you are passing in here doesn't work very well with the component blueprint in this project. It'll have to be implemented, but component blueprints are in a bit of flux at the moment, which I haven't had the time to look properly in to yet.

**Comment by kimroen** [comment]
This should be fixed now in version 1.13.0, which was just released.
